components/dropdown/dropdown-button-subtle.js

```diff
@@ -0,0 +1,37 @@
+import '../button/button-subtle.js';
+import '../icons/icon.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { DropdownOpenerMixin } from './dropdown-opener-mixin.js';
+import { dropdownOpenerStyles } from './dropdown-opener-styles.js';
+
+class DropdownButtonSubtle extends DropdownOpenerMixin(LitElement) {
+
+	static get properties() {
+		return {
+			text: {
+				type: String
+			}
+		};
+	}
+
+	static get styles() {
+		return dropdownOpenerStyles;
+	}
+
+	render() {
+		return html`
+			<d2l-button-subtle text=${this.text} icon="tier1:chevron-down" icon-right=""></d2l-button-subtle>

```

Comment:
`icon-right` is boolean for the component, so we should not need to assign empty string here.